.travis.yml

```diff
@@ -1,5 +1,16 @@
-before_install: bash ./scripts/validate-tests.sh
-after_success: bash ./scripts/docs-upload-gh.sh
+before_install:
+  # Repo for Yarn
+  - sudo apt-key adv --fetch-keys http://dl.yarnpkg.com/debian/pubkey.gpg
```

Comment:
I wonder why `npm i yarn` doesn't work. Seem like some weird hoops to jump through, I'm fine with it if that's what's needed though.